### PR TITLE
7.x newspaper create issues

### PR DESCRIPTION
**Pull request by nigelgbanks**

### CODE REVIEW

File: includes/utilities.inc [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+ *     - issued: A DateTime object repersenting the date the issue was released.
+ */
+function islandora_newspaper_get_issues(FedoraObject $object) {
+  $query = <<<EOQ
+PREFIX islandora-rels-ext: <http://islandora.ca/ontology/relsext#>
+SELECT ?pid ?sequence ?label ?issued
+FROM <#ri>
+WHERE {
+  ?pid islandora-rels-ext:isIssueOf <info:fedora/{$object->id}> ;
+       islandora-rels-ext:isSequenceNumber ?sequence ;
```
Comment (adam-vessey) [inline includes/utilities.inc]: The requirement of the sequence number breaks things ingested via the batch... Add in the sequence number in the batch, or make it optional here?

Comment (nigelgbanks) [inline includes/utilities.inc]: Good catch, I've removed the sequence number as a requirement.

File: theme/theme.inc [hunk 2]
```diff
@@ -5,173 +5,255 @@
...
+/**
+ * Theme the page selector.
+ */
+function theme_islandora_newspaper_page_select(array $variables) {
+  $path = drupal_get_path('module', 'islandora_newspaper');
+  drupal_add_js($path . '/js/islandora_newspaper.js');
+  $object = $variables['object'];
+  $results = $object->relationships->get(ISLANDORA_RELS_EXT_URI, 'isPageOf');
+  $parent = islandora_object_load($results[0]['object']['value']);
+  $pages = islandora_paged_content_get_pages($parent);
```
Comment (adam-vessey) [inline theme/theme.inc:150]: islandora_paged_content_get_pages() may not yet have been loaded... (missing the include)

File: includes/utilities.inc [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+PREFIX islandora-rels-ext: <http://islandora.ca/ontology/relsext#>
+SELECT ?pid ?sequence ?label ?issued
+FROM <#ri>
+WHERE {
+  ?pid islandora-rels-ext:isIssueOf <info:fedora/{$object->id}> ;
+    <fedora-model:label> ?label
+  OPTIONAL {
+    ?pid islandora-rels-ext:dateIssued ?issued ;
+    islandora-rels-ext:isSequenceNumber ?sequence
+  }
```
Comment (adam-vessey) [inline includes/utilities.inc:80]: Need to write this as two separate OPTIONAL sections--missing the sequence number means we don't retrieve the issued date (and vice-versa).

**Pull request merged**